crates/noirc_evaluator/src/ssa_refactor/ir/dfg.rs

```diff
@@ -61,6 +61,9 @@ pub(crate) struct DataFlowGraph {
 
     /// All blocks in a function
     blocks: DenseMap<BasicBlock>,
+
+    /// Brillig compilation artefact of the funtion
+    obj: Artefact,
```

Comment:
We should avoid storing Artifacts for every function since not every function will be compiled to Brillig

Reply:
No, it is precisely here because every function will be compiled to Brillig.

Reply:
Why do we need to compile all functions to Brillig? This seems like it'd be a rare case that all functions actually need to be compiled to Brillig.

Reply:
I compile to brillig only brillig functions for now. Since the PR is for an empty function, we do not need to do more.